### PR TITLE
nodejs-week1

### PR DESCRIPTION
**Pull request by negaren**

### CODE REVIEW

**Comment by shpomp** [comment]
Good job! 🌷 
Nicely structured and commented, well thought out with establishing the needed filtered arrays first.  👍🏽  ✨

**Comment by shpomp** [comment]
@negaren may I ask why is this merged without implementing feedback? 🤓  Especially as one of the tasks is technically not done. ✌🏽

**Comment by negaren** [comment]
Actually I have received feedback from yourself and you have mentioned: Good job! 🌷
Nicely structured and commented, well thought out with establishing the needed filtered arrays first. !

**Comment by negaren** [comment]
@shpomp Actually I have received feedback from yourself and you have mentioned: Good job! 🌷
Nicely structured and commented, well thought out with establishing the needed filtered arrays first. !

**Comment by shpomp** [comment]
I am sorry, @negaren , from your comment I understand you could not see my comments, and that is my fault. I did not realize I have to click to submit review after commenting 🤦🏽‍♂️  I am sorry for thinking you just ignored the comments 😅  ❤️  
I submitted now, but it is not huge things or any huge mistakes at all, so this can stay merged. Just take a look at the comments and consider for the future 🙏🏽

**Comment by negaren** [comment]
@shpomp Kindly let me know which parts need to be revised?

File: nodejs/week1/homework/src/backend/app.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// import data here
+const meals = require("./data/meals.json");
+const reviews = require("./data/reviews.json"); 
+const reservations = require("./data/reservations.json"); 
+
+//meals
+const mealsWithReviews = meals.map(meal => {
+  const mealRev = reviews.filter(review => review.mealId == meal.id);
+  return Object.assign(meal,({"reviews" : mealRev}));
+});
```
[the remark this answers is not in the document]
Comment (negaren) [reply nodejs/week1/homework/src/backend/app.js:13]: Thank @shpomp it was very helpful 🌸

File: nodejs/week1/homework/warmup.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        return 2 * this.radius;
+    }
+    getCircumference(){
+        return 2 * 3.141 * this.radius;
+    }
+    getArea(){
+        return 3.141 * 3.141 * this.radius;
+    }
+}
+
```
[the remark this answers is not in the document]
Comment (negaren) [reply nodejs/week1/homework/warmup.js:18]: That's cool, I will keep in mind @shpomp 🙏🏻

**Comment by shpomp** [comment]
> @shpomp Kindly let me know which parts need to be revised?

for full implementation of the homework, the part of the cheaps meal is technically missing. But it is ok in this case, as you actually attempted something more ambitious than the cheap meals and it is my fault for not submitting the review properly. 🙏🏽  
Consider all things done 👌🏽

**Pull request merged**